prow/cluster/jobs/all-periodics.yaml

```diff
@@ -66,7 +66,7 @@ periodics:
     nodeSelector:
       testing: test-pool
 
-- name: periodic-test-infra-retester
+- name: periodic-retester
```

Comment:
I would call this `periodic-istio-retester` in case you want to add more of these jobs for other repos. You could add more `repo:foo` tags to the query to get the same functionality, but it is probably better to use different jobs for different repos so that the `ceiling` argument can be used to retest a few PRs at a time in each repo instead of possibly retesting PRs from only one of the repos.